spec/controllers/api/v1/replies_controller_spec.rb

```diff
@@ -17,7 +17,7 @@
     end
 
     it "succeeds with valid post", :show_in_doc do
-      post = create(:post, num_replies: 2, with_icon: true, with_character: true)
+      post = create(:post, num_replies: 1, with_icon: true, with_character: true)
```

Comment:
This test suggests we're now showing `post.written` in the API, where we didn't before. Is that correct behavior?

Reply:
I have _no idea_

Reply:
I'm okay with this change, I think. Looks like our current docs don't show the post as being part of the replies endpoint, but it _also_ doesn't show the post content as being part of the _posts_ endpoint, and we need it somewhere.

Should let Vona know though (they maintain glowficrss.com, right?).